mfr/core/extension.py

```diff
@@ -5,7 +5,8 @@
 
 class BaseExporter(metaclass=abc.ABCMeta):
 
-    def __init__(self, source_file_path, output_file_path, format):
+    def __init__(self, ext, source_file_path, output_file_path, format):
```

Comment:
ext and self.ext was used as I saw it already being used elsewhere. Extension in MFR could refer to an actual extension library (eg the images extension, or Unoconv extension) So that is why it was avoided. normalized_name and ext are not actually the same thing. (dont remember why it is this way)

Reply:
Agreed. Let's keep using `ext` then and having the doc string should help for clarification.